public/js/Game.js

```diff
@@ -0,0 +1,107 @@
+ function Game(){
+   this.scoreTotal = 0;
+   this.currentFrame = 1;
+   this.currentRoll = 1;
+   this.lastRollScore = 0;
+   this.pins = 10;
+   this.scoreMode = "normal";
+   this.gameOver = false;
+ }
+
+Game.prototype.hits = function() {
+  return Math.floor(Math.random() * (this.pins + 1));
```

Comment:
It's probably worth passing in each roll (eg 5) rather than generating it inside the game.  Or, if you'd like to randomly generate the roll, it might be worth moving the code out of the Game - it feels like a separate responsibility.